picard/tagger.py

```diff
@@ -308,6 +317,13 @@ def __init__(self, picard_args, unparsed_args, localedir, autoupdate):
         if self.autoupdate_enabled:
             self.updatecheckmanager = UpdateCheckManager(parent=self.window)
 
+    def pipe_server(self):
+        IGNORED = {pipe.Pipe.MESSAGE_TO_IGNORE, pipe.Pipe.NO_RESPONSE_MESSAGE}
+        while self.pipe_running:
+            messages = [x for x in self.pipe_handler.read_from_pipe() if x not in IGNORED]
+            if messages:
+                self.add_files(messages)
```

Comment:
One minor thing I found during testing: This should be `add_paths`  instead of `add_files`, so that calling picard with a directory path also gets processed properly.

Reply:
Thanks!